samples/apps/copilot-chat-app/webapi/DeploymentTemplates/sk.bicep

```diff
@@ -5,8 +5,6 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
 Bicep template for deploying Semantic Kernel to Azure as a web app service.
 
 Resources to add:
-- CosmosDB
```

Comment:
nit: Removing cosmosDB should be part of the other PR, https://github.com/microsoft/semantic-kernel/pull/892/files

Reply:
Changed